server/secondary_indexes.go

```diff
@@ -277,3 +302,92 @@ func (it *secondaryIndexRangeIterator) Value() (*proto.GetResponse, error) {
 
 	return gr, err
 }
+
+// /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
why do we need this?   :)

Reply:
These are just to visually separate different sections